### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by puneethmeister**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ In this example, if you had only published an SPF TXT record for your domain, th
 All the accepted domains of your tenant will be shown in Microsoft 365 Defender portal under DKIM page. If you do not see it, add your accepted domain from [domains page](/microsoft-365/admin/setup/add-domain#add-a-domain).
 Once your domain is added, follow the steps as shown below to configure DKIM.
 
-Step 1: Click on the domain you wish to configure DKIM on DKIM page.
+Step 1: Click on the domain you wish to configure DKIM on DKIM page. https://security.microsoft.com/dkimv2 or https://protection.office.com/dkimv2
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
Step 1: Click on the domain you wish to configure DKIM on DKIM page (https://security.microsoft.com/dkimv2 or https://protection.office.com/dkimv2).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @puneethmeister, kindly commit the minor suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@puneethmeister Gentle reminder.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Review by MSFTTracyP — APPROVED** [review]

**Pull request merged**